test/integration-spec/helper.numerics.js

```diff
@@ -42,67 +40,53 @@ test('[INT][NUMERICS][RAW-API]Test numeric operations on the helper and their re
       var calls = 0;
 
       helper.on('error', function(err) {
-        t.fail(err);
-        t.end();
+        done.fail(err);
       });
 
       helper.on('result', function(content) {
         calls++;
 
         if (calls === 1) {
-          t.equal(content.hits.length, 4, 'No numeric filter: 4 results');
-          t.deepEqual(map(content.hits, hitsToParsedID).sort(),
-            [0, 1, 2, 3],
-            'No tags expected ids: 0, 1, 2, 3');
+          expect(content.hits.length).toBe(4);
+          expect(map(content.hits, hitsToParsedID).sort()).toEqual([0, 1, 2, 3]);
           helper.setQueryParameter('numericFilters', 'n=6,n=10').search();
         }
 
         if (calls === 2) {
-          t.equal(content.hits.length, 1, 'n=6 AND n=10: 1 result');
-          t.deepEqual(map(content.hits, hitsToParsedID).sort(),
-            [1],
-            'n=6 AND n=10 >> expected id: 1');
+          expect(content.hits.length).toBe(1);
+          expect(map(content.hits, hitsToParsedID).sort()).toEqual([1]);
           helper.setQueryParameter('numericFilters', '(n=6,n>40)').search();
         }
 
         if (calls === 3) {
-          t.equal(content.hits.length, 3, 'n=6 OR n>40: 3 results');
-          t.deepEqual(map(content.hits, hitsToParsedID).sort(),
-            [0, 1, 2],
-            'n=6 OR n>40 >> expected ids: 0, 1, 2');
+          expect(content.hits.length).toBe(3);
+          expect(map(content.hits, hitsToParsedID).sort()).toEqual([0, 1, 2]);
           helper.setQueryParameter('numericFilters', 'n:11 to 46').search();
         }
 
         if (calls === 4) {
-          t.equal(content.hits.length, 2, '11 to 46: 2 results');
-          t.deepEqual(map(content.hits, hitsToParsedID).sort(),
-            [2, 3],
-            '11 to 46 >> expected ids: 2, 3');
+          expect(content.hits.length).toBe(2);
+          expect(map(content.hits, hitsToParsedID).sort()).toEqual([2, 3]);
           helper.setQueryParameter('numericFilters', 'n=5,(n=10,n=45)').search();
         }
 
         if (calls === 5) {
```

Comment:
I think this flow can be improved later

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453